src/main/java/com/amazon/opendistroforelasticsearch/security/auditlog/impl/AbstractAuditLog.java

```diff
@@ -122,6 +122,8 @@ protected void onComplianceConfigChanged(ComplianceConfig complianceConfig) {
         this.complianceConfig = complianceConfig;
         enableRoutes();
         this.complianceConfig.log(log);
+        // External config is audit logged only once per node start and config from index is not available at that time.
+        // The audit event will be created only if enabled and not already logged.
```

Comment:
And externalConfig parameter can be hot-reloadable which will impact functionality of "logExternalConfig()"